src/Raven.Server/Documents/Indexes/MapReduce/Static/MapReduceIndex.cs

```diff
@@ -30,6 +30,8 @@ public class MapReduceIndex : MapReduceIndexBase<MapReduceIndexDefinition>
 
         private readonly Dictionary<string, AnonymousObjectToBlittableMapResultsEnumerableWrapper> _enumerationWrappers = new Dictionary<string, AnonymousObjectToBlittableMapResultsEnumerableWrapper>();
 
+        public PropertyAccessor OutputReduceToCollectionPropertyAccessor;
```

Comment:
As it is not related to the Lucene persistence it feel more natural to put this on the index itself (map-reduce only).